ludwig/utils/trainer_utils.py

```diff
@@ -210,6 +212,7 @@ def log_metrics(self):
             "best_valid_metric": self.best_eval_metric_value,
             "num_reductions_lr": self.num_reductions_learning_rate,
             "num_increases_bs": self.num_increases_batch_size,
+            "llm_eval_examples": self.llm_eval_examples,
```

Comment:
Maybe add a comment here too about how the structure of llm_eval_examples is different from all other entries and certain callbacks.

> Note that `llm_eval_examples` is the only entry in the map with string leaves. Loggers and other callbacks should handle this accordingly.